source/material_model/rheology/frank_kamenetskii.cc

```diff
@@ -41,11 +41,25 @@ namespace aspect
       template <int dim>
       double
       FrankKamenetskii<dim>::compute_viscosity (const double temperature,
+                                                const double pressure,
+                                                const double density,
                                                 const unsigned int composition) const
       {
         const double reference_temperature = this->get_adiabatic_surface_temperature();
+        const double reference_pressure = this->get_surface_pressure();
+        const double max_depth = this->get_geometry_model().maximal_depth();
+
+        // get the magnitude of gravitational acceleration
```

Comment:
You might be able to simplify this as follows:
1. Add `#include <aspect/gravity_model/interface.h>` at the top of this file.
2. Then use `const double gravity_norm = this->get_gravity_model().gravity_vector(in.position[0]).norm();` below.